src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -34,4 +36,39 @@ private ModItems()
          * Intentionally left empty.
          */
     }
-}
+
+    /**
+     * Initates all the blocks. At the correct time.
+     * @param registry
+     */
+    public static void init(final IForgeRegistry<Item> registry)
+    {
+        supplyChest = new ItemSupplyChestDeployer();
+        buildTool = new ItemBuildTool();
+        scanTool = new ItemScanTool();
+        permTool = new ItemScepterPermission();
+        caliper = new ItemCaliper();
+        scepterGuard = new ItemScepterGuard();
+        supplyCamp = new ItemSupplyCampDeployer();
+        ancientTome = new ItemAncientTome();
+        chiefSword = new ItemChiefSword();
+        itemAchievementProxySettlement = new ItemAchievementProxy("sizeSettlement");
+        itemAchievementProxyTown       = new ItemAchievementProxy("sizeTown");
+        itemAchievementProxyCity       = new ItemAchievementProxy("sizeCity");
+        itemAchievementProxyMetropolis = new ItemAchievementProxy("sizeMetropolis");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'itemAchievementProxyMetropolis' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2444)